src/log/log.c

```diff
@@ -2272,7 +2277,7 @@ __wt_log_scan(WT_SESSION_IMPL *session, WT_LSN *lsnp, uint32_t flags,
 				    PRIu32 "/%" PRIu32,
 				    lsnp->l.file, lsnp->l.offset);
 			else
-				return (WT_NOTFOUND);
+				WT_ERR (WT_NOTFOUND);
```

Comment:
The change is fine as long as we're leaving the function and the caller gets `WT_NOTFOUND` here. 

Minor nit: Please remove the space to be: `WT_ERR(...)`

Reply:
Fixed in 5251f40.